src/App.tsx

```diff
@@ -1,21 +1,19 @@
-
 import "./App.css";
-import { BooksPage } from "./domain/books/BooksPage";
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
-import { CartPage } from "./domain/cart/CartPage";
 import Navbar from "./components/Navbar/Navbar";
+import { routes } from "./routes/routes";
 
 function App() {
-
   return (
     <Router>
       <Navbar />
       <Routes>
-        <Route path="/" element={<BooksPage />} />
-        <Route path="/cart" element={<CartPage />} />
+        {routes.map(({ path, Component }) => (
+          <Route path={path} element={<Component />} />
+        ))}
```

Comment:
I think you have not understood what I was saying.
In, src/router/router.tsx file, you should be placing the routes.

`export const router = () => {
  <Routes>
   <Route path="/" component={<BooksPage />} />
   </Routes>
|
`

Reply:
Okay

Reply:
Fixed